sfdx-source/apex-common-samplecode/main/classes/selectors/PricebookEntriesSelector.cls

```diff
@@ -53,13 +53,6 @@ public class PricebookEntriesSelector extends fflib_SObjectSelector
 
 	public List<PricebookEntry> selectById(Set<ID> idSet)
 	{
-		assertIsAccessible();
-		String query = String.format('select {0},{3},{4} FROM {1} WHERE id in :idSet order by {2}', 
-			new List<String>{
-				getFieldListString(),
-				getSObjectName(),
-				getOrderBy()
-			});
-		return (List<PricebookEntry>) Database.query(query);
+		return (List<PricebookEntry>) selectSObjectsById(idSet);
```

Comment:
Why does this method even exist? WHy not direclty call `selectSObjectsById()` from client code?

Reply:
The output of these `selectById` methods in each selector class allows for the casting of the result list to the appropriate SObjectType.  The `selectSObjectsById` method only returns `List<SObject>`.

Reply:
fine fine